components/eventing-controller/pkg/backend/eventmesh/eventmesh.go

```diff
@@ -254,9 +215,97 @@ func (em *EventMesh) GetProcessedEventTypes(kymaSubscription *eventingv1alpha2.S
 	return result, nil
 }
 
+// handleKymaSubModified checks if the Kyma subscription is modified.
+// If modified, then it deletes the corresponding subscription on EventMesh and returns true.
+func (em *EventMesh) handleKymaSubModified(eventMeshSub *types.Subscription, kymaSub *eventingv1alpha2.Subscription) (bool, error) {
+	isEventMeshSubModified, err := backendutils.IsEventMeshSubModified(eventMeshSub, kymaSub.Status.Backend.Ev2hash)
```

Comment:
(nit) the naming is a little tricky here. The function says `kyma sub` but inside we do `eventmesh sub` and the next function below is `eventmesh sub`

Reply:
updated